app/schemas.py

```diff
@@ -105,9 +105,25 @@ class PriceCreate(BaseModel):
         This ID must be a canonical category ID in the Open Food Facts taxonomy.
         If the ID is not valid, the price will be rejected.""",
     )
+    labels_tags: list[str] | None = Field(
```

Comment:
possible to add a regex pattern for `list[str]` fields ? :sweat_smile:

Reply:
Not really!
It could be added in the validator, but it's not really necessary, as we check that the label tag is in the taxonomy